### DISCUSSION TITLE
problemas con mi cuenta

### DISCUSSION THREAD

**Discussion by LeaBertero** [community/community#72177] [Accessibility]
### Select Topic Area

Question

### Body

al parecer no se pueden visualizar ni mis repositorios ni mi perfil de ususario cuando intento compartir con otras personas, les arroja un error 404 cuando intentan acceder. como puedo brindar una solucion a este problema, muchas gracias ! necesito una solucion urgente, para poder autogestionarme coon github, gracias !

**Comment by LeaaBertero** [comment]
Buenas tardes, solicito ayuda para poder ingresar a mi cuenta LeaBertero con la cuenta de hotmail asciada , ya que no puedo ingresar por el motivo de que me solicita un numero de autenticacion de 6 digitos, el cual no cuento, por que he extraviado y no se donde ni como hacer para adquirirlo, no se cuales son los motivos por lo cual no cuento con esa clave de autenticacion que me solicita, por favor, necesit que me ayuden para poder ingresar con esa clave de autentocacion , por que es mi cuenta principal y la necesito con urgencia, muchas gracias , espero que puedan ayudarme.

**Reply by ghostinhershell** [reply]
Hi @LeaaBertero,

I've removed your private information from [this post](link to discussion) as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).

**Comment by ghostinhershell** [comment]
Hi @LeaBertero, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.

**Comment by LeaaBertero** [comment]
Good afternoon, I am requesting help to be able to enter my LeaBertero account with the associated hotmail account, since I cannot enter because it asks me for a 6-digit authentication number, which I do not have, because I have lost it and not I don't know where or how to acquire it, I don't know the reasons why I don't have that authentication key that you are asking for, please, I need help to be able to log in with that authentication key, because it is my main account and I need it urgently, thank you very much, I hope you can help me.

**Reply by ghostinhershell** [reply]
Gotcha, thanks for clarifying! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
